pkg/util/clouderrors/errors.go

```diff
@@ -0,0 +1,28 @@
+package clouderrors
```

Comment:
This package stinks. But there is few things I dont want to do:
1. I dont want to add this into `util/aad` package as all other package will copy paste this pattern.
2. We will have more of these when dynamic gets split.
3. I dont want to add this into API (but less against than point 1)

in AAD Example this code is being used in 2 packages, so if we chose to add this `switch` statement close to the caller, it will be copy paste 2x. And in time more places in the code will have these complicated case statements (in validator case it will be 3-4 cases....